graphql/src/query.rs

```diff
@@ -18,6 +18,18 @@ pub struct Query;
 
 #[async_graphql::Object]
 impl Query {
+    // query for 1 collection based on slug, and make a collection object with all the stuff in it.
+    async fn get_edited_collection(
```

Comment:
Let's call this `edited_collection`, we always use noun phrases for GraphQL fields (rather than verbs like with normal functions).